### PR TITLE
[ADD] National Id Number (Matricule) field to the company

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**

### CODE REVIEW

**Comment by lmignon** [comment]
Does this need is specific to Luxembourg?

**Comment by sbidoul** [comment]
It would be nice to hide this field if the company is not a luxembourg company.

**Comment by Cedric-Pigeon** [comment]
@sbidoul Request done.

File: l10n_lu_ext/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+##############################################################################
+
+from openerp import models, fields
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.company"
+
+    matricule = fields.Char(string='National Id Number', size=13)
```
Comment (sbidoul) [inline l10n_lu_ext/models/res_company.py]: @Olivier-LAURENT do you have a small help text explaining how/where this number can be obtained?

Comment (Olivier-LAURENT) [inline l10n_lu_ext/models/res_company.py]: Tax Identification Number delivered by the authorities as soon as the company is registered

**Comment by sbidoul** [comment]
:+1: 

@lmignon this is indeed Luxemburg specific

**Comment by dreispt** [comment]
Thinking out loud: it would be desirable to ensure that l10n specific fields are unique, in order to avoid conflicts in case we need to have several localizations installed in the same database. Thoughts?

**Comment by sbidoul** [comment]
@dreispt I've no idea how to ensure that. I'd say this issue is not specific to localizations as any module can potentially generate naming conflicts.

**Comment by lmignon** [comment]
:+1:

**Comment by yvaucher** [comment]
On option would be to simply rename matricule to l10n_lu_matricule for exemple.

Otherwise matricule shouldn't be restricted by size but would need a python constraint that check country as well. This would make the need of a base module adding matricule that could be inherited by other location.

The questions are have we any other location that requires matricule? And do we need multiple matricules on a single company

**Comment by sbidoul** [comment]
@dreispt @yvaucher I renamed matricule to l10n_lu_matricule.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**